net-misc/ubridge/files/ubridge-0.9.14-respect-flags.patch

```diff
@@ -0,0 +1,20 @@
+--- a/Makefile	2018-03-16 09:15:10.528245830 +0500
++++ b/Makefile	2018-03-16 09:16:09.588132065 +0500
+@@ -40,7 +40,7 @@
+ 
+ CC      ?=   gcc
```

Comment:
You need to explicitly write `src_compile` and call it as `emake CC="$(tc-getCC)"` (from toolchain-funcs) to respect CC correctly.